js/ui/sortable.js

```diff
@@ -17,6 +17,23 @@ const CLONE_CLASS = 'clone';
 
 const isElementVisible = itemElement => $(itemElement).is(':visible');
 
+const animate = (element, config) => {
+    if(!element) return;
+
+    const left = config.to?.left || 0;
+    const top = config.to?.top || 0;
+
+    element.style.transform = `translate(${left}px,${top}px)`;
+    element.style.transition = fx.off ? '' : `all ${config.duration}ms ${config.easing}`;
```

Comment:
### This is wrong

In order to animate elements, the transition setting needs to remain constant.

You must only change the CSS properties that should be transitioned from value X to value Y. 

### Problems
#### 1. Incorrect Animation Appearance
When value Y is the base state (e.g. no translation), you must not clear the transition property (like in line 34), because that avoids the "reverting" animation from happening.
This can be seen with your code applied, where the rows in a 2nd grid immediately jump back instead of closing the gap in an animated way.

#### 2. Performance
Another reason not to change the transition via style and for each element individually is browser performance. (You should not only test this with 100 rows in Chrome)

Best performance will be achieved by assigning this via a CSS rule to all rows and never add or remove that CSS class from items.

#### 3. Limit Animation Properties

Specifying `all` for the transition is not a good way, because it will animate all style changes which is surely undesired. 

Use 

`transform ${config.duration}ms ${config.easing}`;

instead.